settings.py

```diff
@@ -25,4 +25,8 @@
 
 INSTALLED_APPS = [
     "db",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
```

Comment:
Could you clarify the purpose? I'm not sure I understand.

Reply:
I had it like one of tasks in README.md